client/rpcserver/handlers.go

```diff
@@ -582,16 +581,18 @@ var helpMsgs = map[string]helpMsg{
 	},
 	newWalletRoute: {
 		pwArgsShort: `"appPass" "walletPass"`,
-		argsShort:   `assetID "account" ("config")`,
+		argsShort:   `assetID ("path" "settings")`,
 		cmdSummary:  `Connect to a new wallet.`,
 		pwArgsLong: `Password Args:
     appPass (string): The DEX client password.
     walletPass (string): The wallet's password.`,
 		argsLong: `Args:
     assetID (int): The asset's BIP-44 registered coin index. e.g. 42 for DCR.
       See https://github.com/satoshilabs/slips/blob/master/slip-0044.md
-    account (string): The account or wallet name, depending on wallet software.
-    config (string): Optional. The path to the wallet's config file.`,
+     path (string): Optional. The path to a configuration file.
+     settings (string): A JSON-encoded string->string mapping of additional
+       configuration settings. These settings take precedence over any settings
+       parsed from file. e.g. '{"account":"default"}'`,
```

Comment:
I had to dig into btc.go's definition of it's `configOpts` to locate the `walletname` key for bitcoin, so could we also put `'{"walletname":"gamma"}'` as an example?  Maybe `e.g. '{"account":"default"}' for Decred accounts, and '{"walletname":""}' for the default Bitcoin wallet where bitcoind's listwallets RPC gives possible walletnames`